doc-content/shared-kie-docs/src/main/asciidoc/Commands/runtime-commands-con.adoc

```diff
@@ -0,0 +1,21 @@
+[id='runtime-commands-con_{context}']
+= Runtime commands in {PRODUCT}
+
+*<@Maciej: Compare this chapter (validated by Matteo Mortari) with <<kie-server-commands-con_kie-apis>>. They're slightly different. Do both chapters make sense and are both correct from your perspective?>*
```

Comment:
yes, they both are correct and valid. The only comment I have is about order of these. I would put the server commands first and then drools runtime commands after it

Reply:
@mswiderski Okay. Can you please see my two other comments in the KIE Server commands chapter? Those two are the more critical ones. Refresh the doc preview and search *@Maciej* and you will find 3 comments total, this one in chap. 3 and the other two in chap. 4.